classes/RequestSql.php

```diff
@@ -310,7 +310,9 @@ public function returnNameTable($alias, $tables, $attr = null)
                 if (!isset($table['alias']) || !isset($table['table'])) {
                     continue;
                 }
-                if ($table['alias']['no_quotes'] == $alias || $table['alias']['no_quotes']['parts'][0] == $alias) {
+                /** @var array{'parts': array<int, bool>} $tableAlias */
```

Comment:
```suggestion
                /** @var array{'parts': array<int, bool>}|bool $tableAlias */
```

Seems like it can also be a boolean no, since it's compared with `$alias`?

Reply:
I try but PHPStan returns this error :

```
 ------ ---------------------------------------- 
  Line   RequestSql.php                          
 ------ ---------------------------------------- 
  315    Cannot access offset 'parts' on false.  
 ------ ---------------------------------------- 
```

Reply:
Hmm, actually there is another more important issue `$alias` can also be a string (see L276)
So the input parameter should be `@param string|false $alias`

And then you can type like that:
`/** @var array{'parts': array<int, bool>}|string $tableAlias */`

Which seems to make more sense

Reply:
Fixed. Thanks.